doc/manual/manual.tex

```diff
@@ -5365,6 +5365,91 @@ \subsubsection{Tracking finite strain}
+to convert a drawing generated with the vector graphics editor Inkscape (\url{https://inkscape.org/en/}) to a data 
+file that can be read into \aspect{}. Here, we will demonstrate how this can be done for a 2d setup for a model
+with one compositional field, but geomIO also has the capability to create 3D volumes based on series of 2D vector 
+drawings using any number of different materials; and the initial conditions could also be used with particles 
+instead of compositional fields. 
+
+To obtain the developer version of geomIO, you can clone the bitbucket repository by executing the command
+\begin{verbatim}
+ git clone https://bitbucket.org/geomio/geomio.git
+\end{verbatim}
+or you can download geomIO \href{https://bitbucket.org/geomio/geomio/downloads}{here}. 
+You will then need to add the geomIO source folders to your MATLAB path by running the file located in
+\texttt{/path/to/geomio/installation/InstallGeomIO.m}.
+An extensive documentation for how to use geomIO can be found \href{http://geomio-doc.bitbucket.org/}{here}.
+Among other things, it explains \href{http://geomio-doc.bitbucket.org/tuto2D.html#drawing}{how to generate drawings in Inkscape}
+that can be read in by geomIO, which involves assigning new attributes to paths in Inkscape's XML editor. 
+In particular, a new property `phase' has to be added to each path, and set to a value corresponding to the 
+index of the material that should be present in this region in the initial condition of the geodynamic model.   
+
+\note{geomIO currently only supports the latest stable version of Inkscape (0.91), and other versions might not
+  work with geomIO or cause errors. Moreover, geomIO currently does not support grouping paths (paths can still 
+  be combined using \texttt{Path$\rightarrow$Union}, \texttt{Path$\rightarrow$Difference} or similar commands), 
+  and only the outermost closed contour of a path will be considered. This means that i.e. for modelling a spherical 
+  annulus, you would have to draw two circles, and assign the inner one the same phase as the background of your drawing.}
+
+We will here use a drawing of a jellyfish located in \url{doc/manual/cookbooks/geomio/jellyfish.svg}, where different phases 
+have already been assigned to each path (Figure \ref{fig:jelly-picture}). 
+\begin{figure}[h!]
+    \centering
+    \includegraphics[width=0.2\textwidth]{cookbooks/geomio/jellyfish.pdf}
+    \caption{Vector drawing of a jellyfish.}
+    \label{fig:jelly-picture}
+\end{figure}
+\note{The page of your drawing in Inkscape should already have the extents (in px) that you later want to use in your model (in m).}
+
+After geomIO is initialized in MATLAB, we \href{http://geomio-doc.bitbucket.org/tuto2D.html#assigning-phase-to-markers}
+{run geomIO as desribed in the documentation}, loading the default options and then specifying all the option we want to 
+change, such as the path to the Input file, or the resolution. 
```

Comment:
maybe put a colon instead of a period at the end of the sentence, since it prefixes the listing in the next line.